src/sparseml/pytorch/image_classification/export.py

```diff
@@ -290,6 +297,13 @@ class strings. Expected to be a path to a
     show_default=True,
     help="number of classes for model; must be set if a dataset is not provided",
 )
+@click.option(
+    "--one-shot",
+    "--one_shot",
+    is_flag=True,
+    show_default=True,
```

Comment:
missing default value?

Reply:
Flag always has a default value of `False`